osmnx/settings.py

```diff
@@ -180,3 +184,152 @@
     "tunnel",
     "width",
 ]
+implicit_maxspeed_values = {
```

Comment:
My one main comment is that I don't think this should be a user configurable setting, but rather a constant in the `routing` module. This would make it consistent with how [we handle](https://github.com/gboeing/osmnx/blob/d614c9608165a614f796c7ab8e57257e7f9f1a63/osmnx/features.py#L49) the _POLYGON_FEATURES constant in the `features` module, for example. That is, if we scrape some defined values from OSM/Overpass and hardcode them, we should make them a constant and update them if the data source changes in the future. Because this is Python, there's the added benefit that a user can access and update these "private" variables if they wish to. For instance, `ox.features._POLYGON_FEATURES = {"new_keys": "new_values"}`. Let's handle these implicit maxspeed values the same way in the `routing` module.